### ISSUE TITLE
Create import tool for buildings on voiceofsap.org

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#558]
This doesn't have a lot of buildings and most of them are in India but it has a single accessibility rating for each building which could be useful for AccessLocator.

Voice of Special Abled People(SAP) rates buildings for accessibility in an effort to promote improvements to accessibility and visibility of people with special needs.

https://www.voiceofsap.org/buildings/

**Comment by M4ttoF** [comment]
After looking at this site for a bit it seems like it would be troublesome to use for information. On pages with accessibility information it will only say which services ARENT available. So you'll just be able to know where you cant go rather than where you can go. For the places that are high rated it just says "Perfect" but I'm not sure what all that implies.
